projects/pygments.org/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: https://github.com/pygments/pygments/archive/refs/tags/{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: pygments/pygments/tags
+
+dependencies:
+  python.org: 3.11
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    
+  script: |
+    python-venv.sh {{prefix}}/bin/pygmentize
+
+provides:
+  - /bin/pygmentize
```

Comment:
the `/` prefix will evade our magic

Reply:
Certainly need some CI checks for this sort of thing

Reply:
woah good catch. I don't know how that got that...